pghoard-local-minimal.json

```diff
@@ -13,7 +13,7 @@
             "object_storage": {
                 "storage_type": "local",
                 "directory": "./backups"
-            }
+            },
```

Comment:
Thanks for fixing this, mind removing the extra comma from the following line, too?